spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc

```diff
@@ -92,12 +70,9 @@ spring.sleuth.web.skipPattern=(^cleanup.*|.+favicon.*)  # Ignore some URL paths.
 Spring Cloud GCP Trace does override some Sleuth configurations:
 
 - Always uses 128-bit Trace IDs. This is required by Stackdriver Trace.
-- Does not use Span joins. Span joins will share the span ID between the client and server Spans.
-  Stackdriver requires that every Span ID within a Trace to be unique, so Span joins are not supported.
-- Uses `StackdriverHttpClientParser` and `StackdriverHttpServerParser` by default to populate Stackdriver
-  related fields.
+- Does not use Span joins. Span joins will share the span ID between the client and server Spans. Stackdriver requires that every Span ID within a Trace to be unique, so Span joins are not supported.
```

Comment:
maybe make it hard-fail (by auto-wiring the property for sharing span ID and 128-bit ID and checking them when producing the sender bean)

Reply:
We're already forcing 128 bit trace IDs and not supporting trace joins in autoconfiguration.

```
@Bean
@Primary
public SleuthProperties stackdriverSleuthProperties(SleuthProperties sleuthProperties) {
	sleuthProperties.setSupportsJoin(false);
	sleuthProperties.setTraceId128(true);
	return sleuthProperties;
}
```